torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2012,7 +2012,10 @@ def sample_inputs_binary_pwise(
         rhs_make_tensor_kwargs=rhs_make_tensor_kwargs,
     )
 
-    scalar = make_arg((), **rhs_make_tensor_kwargs)
+    scalar_kwargs = copy.copy(rhs_make_tensor_kwargs)
```

Comment:
Note this will be a merge conflict with https://github.com/pytorch/pytorch/pull/74280, so whoever merges second will need to update their PR